pyHS100/smartdevice.py

```diff
@@ -406,20 +441,24 @@ def get_emeter_daily(self,
 
         response = self._query_helper(self.emeter_type, "get_daystat",
                                       {'month': month, 'year': year})
+        response = [EmeterStatus(**x) for x in response["day_list"]]
 
-        if self.emeter_units:
-            key = 'energy_wh'
-        else:
+        key = 'energy_wh'
+        if kwh:
```

Comment:
Do you think this is an overcomplication of the API?  Since this PR intends to simply fix the current incompatibility between the hardware/firmware versions, I don't think it's worthwhile to introduce this just yet.

I expect for 99% of consumers, `kwh` will be the default - `True`. If a consumer wanted `wh` only, then I think that should be their concern to do the multiplication within their application.

Reply:
I'm not sure I understand what you propose. Removing the parameter (which defaults to True for kwh)? It just got to that by doing this PR iteratively, and I don't see it complicating it that much.

Reply:
Sounds good to leave, i'm not a maintainer of this repo, but just thought i'd voice my opinion on this. I think it's ok to leave it as it seems it was discussed earlier in the PR.